<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerDialog</class>
 <widget class="QDialog" name="PlayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlayerDialog</string>
  </property>
  <widget class="QWidget" name="wdg_content" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>721</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="MyOpenGLWidget" name="wdg_show" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lb_curTime">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>63</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_endTime">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>500</y>
     <width>63</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_process">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>500</y>
     <width>571</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>539</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pb_start">
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_resume">
      <property name="text">
       <string>播放</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_pause">
      <property name="text">
       <string>暂停</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_stop">
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>298</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyOpenGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">myopenglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
